### DISCUSSION TITLE
为什么明明学生认证成功了，没过几天就突然让我重新认证

### DISCUSSION THREAD

**Discussion by Morolis** [community/community#50195] [GitHub Education]
前几天刚刚认证成功的学生认证，没过几天突然就要重新认证

**Comment by samus-aran** [comment]
Hi @MoonforDream , thanks for participating!
We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
